app/content/assignments/index.js

```diff
@@ -49,6 +49,22 @@ const AssignmentList = () => {
     );
   }
 
+  // Handle both errors of 'cannot read property "reduce" of undefined' and
```

Comment:
This code is already existing in this file at line ~36. Is there a reason for this change?

Also I think there is a "login fixes" commit in this PR that is already in layout branch?